pandas/tests/test_frame.py

```diff
@@ -3568,6 +3565,20 @@ def test_constructor_tuples(self):
         expected = DataFrame({'A': Series([(1, 2), (3, 4)])})
         assert_frame_equal(result, expected)
 
+    def test_constructor_namedtuples(self):
+        # GH11181
+        from collections import namedtuple
+        named_tuple = namedtuple("Pandas", list('ab'))
+        tuples = [named_tuple(1, 3), named_tuple(2, 4)]
+        expected = DataFrame({'a': [1, 2], 'b': [3, 4]})
+        result = DataFrame(tuples)
+        assert_frame_equal(result, expected)
+
```

Comment:
Yes, will do now.

To be clear, is the case you're suggesting: `DataFrame` receives a list of `namedtuple`s, each with the same number of items, but with different `_fields`?
The intended outcome here is that it takes `_fields` from the first. Is that the outcome you want to test for?